### PR TITLE
bump cromwell artifacts versions and dependencies

### PR DESCRIPTION
**Pull request by wshands**
**Description**
Bump the webservice cromwell version and artifacts and dependencies to version 77

**Issue**
https://ucsc-cgl.atlassian.net/browse/DOCK-2084

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that you pass the basic style checks and unit tests by running `mvn clean install`
- [x] Follow the existing JPA patterns for queries, using named parameters, to avoid SQL injection
- [x] Check the Snyk dashboard to ensure you are not introducing new high/critical vulnerabilities
- [x] Assume that inputs to the API can be malicious, and sanitize and/or check for Denial of Service type values, e.g., massive sizes
- [x] Do not serve user-uploaded binary images through the Dockstore API
- [x] Ensure that endpoints that only allow privileged access enforce that with the `@RolesAllowed` annotation
- [x] Do not create cookies, although this may change in the future

### CODE REVIEW

**Review by Richard-Hansen — APPROVED** [review]
Will need to keep an eye on the CLI to see if new cromwell artifacts cause any test breakage.

**Review by denis-yuen — APPROVED** [review]

**Review by coverbeck — APPROVED** [review]
Pending resolution of Denis' comment.

**Review by NatalieEO — APPROVED** [review]

File: dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/QuayImageRegistryTest.java [hunk 1]
```diff
@@ -25,7 +26,7 @@
      * This tests that when there are over 500 tags, we can use the paginated endpoint to retrieve all of them instead.
      * Using calico/node because it has over 3838 tags
      */
-    //    @Ignore("https://github.com/dockstore/dockstore/issues/4663")
+    @Ignore("https://github.com/dockstore/dockstore/issues/4663")
```
[the remark this answers is not in the document]
Comment (coverbeck) [reply dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/QuayImageRegistryTest.java:29]: @wshands You didn't address this comment.

[the remark this answers is not in the document]
Comment (wshands) [reply dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/QuayImageRegistryTest.java:29]: Still seems broken https://github.com/dockstore/dockstore/pull/4780?

**Pull request merged**